scripts/ci/cmake/ci-debian-sid.cmake

```diff
@@ -0,0 +1,13 @@
+# Client maintainer: chuck.atkins@kitware.com
+
+set(dashboard_cache "
+ADIOS2_USE_EXTERNAL_DEPENDENCIES=ON
+ADIOS2_USE_EXTERNAL_GTKORVO_STACK=OFF
```

Comment:
This should still turn on the explicit features, otherwise a failure in auto-detection would successfully configure and build a configuration we don't expect.